src/storage.js

```diff
@@ -0,0 +1,44 @@
+// Mimic localStorage API with chrome.storage.
+// See also: https://developer.chrome.com/docs/extensions/reference/storage/
+const storage = {
+    getAllItems: () => new Promise(resolve => {
```

Comment:
This method is currently unused, but maybe you intended for it to be used by the search-extensions themselves?

Reply:
Yes, exactly.